middleware/cors.js

```diff
@@ -13,6 +13,19 @@ var cors = require('cors');
 
 exports.init = function(app, config, logger, callback) {
     var cfg = config.get('cors');
+    if (typeof cfg.origin === 'object') { //this is meant to be regex
```

Comment:
`cors.origin` can be an array ... I don't think this gets handled here ... an array is an 'object' type

If there is already a tests that works for this, please point it out.
Otherwise, let's add one.

e.g.:
```json
  "cors": {
    "origin": [
      "http://localhost:10080",
      "http://localhost:10081",
      "http://localhost:10082"
    ],
    "exposedHeaders": [
      "X-Update-Token"
    ],
    "credentials": true
  }
```